### PR TITLE
Update with aucdi latest mappings

### PR DESCRIPTION
**Pull request by dt-r**
Updating AUeReqDI Mapping page with the latest AUCDI Mapping content.

### CODE REVIEW

File: input/pagecontent/auereqdi.md [hunk 1]
```diff
@@ -276,9 +276,9 @@ Column attribute descriptions are as follows:
   </tr>
   <tr>
     <td colspan="2">Last updated</td>
-    <td>TBD</td>
-    <td>TBD</td>
-    <td>This data element is added to AUCDI R1; work is underway to map to AU Core.</td>
+    <td>-</td>
+    <td>-</td>
+    <td>This AUCDI element is a <a href="future.html#future-candidate-requirements-under-consideration">Future Candidate Requirement Under Consideration</a>. Future versions of AU Core will develop and test approaches to addressing Last Updated.</td>
```
Comment (dbojicic) [inline input/pagecontent/auereqdi.md:281]: Broken link; should link to AU Core [Future Candidate Requirements Under Consideration](https://build.fhir.org/ig/hl7au/au-fhir-core/future.html#future-candidate-requirements-under-consideration).

File: input/pagecontent/auereqdi.md [hunk 2]
```diff
@@ -307,9 +307,9 @@ Column attribute descriptions are as follows:
...
```
Comment (dbojicic) [inline input/pagecontent/auereqdi.md:312]: Broken link; should link to AU Core [Future Candidate Requirements Under Consideration](https://build.fhir.org/ig/hl7au/au-fhir-core/future.html#future-candidate-requirements-under-consideration).

File: input/pagecontent/auereqdi.md [hunk 3]
```diff
@@ -332,9 +332,9 @@ Column attribute descriptions are as follows:
...
```
Comment (dbojicic) [inline input/pagecontent/auereqdi.md:337]: Broken link; should link to AU Core [Future Candidate Requirements Under Consideration](https://build.fhir.org/ig/hl7au/au-fhir-core/future.html#future-candidate-requirements-under-consideration).

**Review by dbojicic — CHANGES_REQUESTED** [review]

File: input/pagecontent/auereqdi.md (hunk 1 above)
Comment (dt-r) [reply input/pagecontent/auereqdi.md]: fixed the three instances.

File: input/pagecontent/auereqdi.md [hunk 4]
```diff
@@ -332,9 +332,9 @@ Column attribute descriptions are as follows:
   </tr>
   <tr>
     <td colspan="2">Last updated</td>
-    <td>TBD</td>
-    <td>TBD</td>
-    <td>This data element is added to AUCDI R1; work is underway to map to AU Core.</td>
+    <td>-</td>
+    <td>-</td>
+    <td>This AUCDI element is a <a href="https://build.fhir.org/ig/hl7au/au-fhir-core/future.html#future-candidate-requirements-under-consideration">Future Candidate Requirement Under Consideration</a>. Future versions of AU Core will develop and test approaches to addressing Last Updated.</td>
```
Comment (dbojicic) [inline input/pagecontent/auereqdi.md:337]: Not the right spot in the file - commenting here to get an inline comment. 

"Person Recorded Sex or Gender extension" link is broken, please fix.

**Review by dbojicic — CHANGES_REQUESTED** [review]
One small change needed.

File: input/pagecontent/auereqdi.md (hunk 4 above)
Comment (dt-r) [reply input/pagecontent/auereqdi.md:337]: fixed

**Review by dbojicic — APPROVED** [review]

**Pull request merged**